integration-test/gradlew.bat

```diff
@@ -1,3 +1,19 @@
+@rem
```

Comment:
1. This file is not fitted with CRLF linebreaks (I assume it should be, since this is Windows script)
2. Are there any reasons that root `gradlew.bat` was not updated?